tests/AbstractGelfTest.php

```diff
@@ -20,22 +20,22 @@ protected function checkResults(array $events): void
             self::assertTrue(!empty($event['host']) || ($event['short_message'] === 'No host'));
             $timestamps[] = $event['timestamp'];
             $hosts[] = empty($event['host']) ? 'empty' : $event['host'];
-            if (!empty($event['file'])) {
-                $file = $event['file'];
+            if (!empty($event['_file'])) {
```

Comment:
Tohle se zmenilo tim updatem GELFu? Je to v pohode? :thinking:

Reply:
jj, jestli je to definitivně v pohodě budu řešit ještě na testingu, ale zatím jsem nenarazil na problém no, ale takjako jistej si tím taky nejsem na 100%

Reply:
ok, tak uvidime na testingu

Reply:
Zkoušel jsem různý komponenty a nenašel jsem žádný rozdíly, takže to jdu mergnout